octoprint_mrbeam/migrate.py

```diff
@@ -817,3 +818,9 @@ def _set(path, _data, set_func, fullpath=None):
             _set(path, data, settings().setBoolean)
         settings().save()
         self._logger.info("Done.")
+
+    def track_devpi(self):
```

Comment:
- needs to be called
- add try/catch
- add log output indication that this is about to happen